api/Permissions.json

```diff
@@ -257,9 +257,6 @@
       "permission_clipboard-read": {
         "__compat": {
           "description": "<code>clipboard-read</code> permission",
-          "tags": [
-            "web-features:async-clipboard"
-          ],
```

Comment:
> The "clipboard-read" permission was removed from the spec:
> [w3c/clipboard-apis#132](https://github.com/w3c/clipboard-apis/pull/132)

If this is no longer part of the spec, then should this feature also be `standards_track: false`?

Reply:
Done in https://github.com/mdn/browser-compat-data/pull/23714.